core/pkg/monitor/apple/winibuild.py

```diff
@@ -1,15 +1,22 @@
 """Builds the AppleStats Swift binary for monitoring system metrics."""
 
 import pathlib
-import platform
 
-from tools.wini import subprocess
+from tools.wini import arch, subprocess
 
 
-def build_applestats(output_path: pathlib.PurePath) -> None:
+def build_applestats(
+    architecture: arch.Arch,
+    output_path: pathlib.PurePath,
+) -> None:
     """Builds the AppleStats Swift binary.
 
+    NOTE: Swift creates a cache in a directory called ".build/" which speeds
```

Comment:
do you think we should add a clean option, that just remove this preemptively?

Reply:
yes, probably a good idea! i will do it in a separate PR though